invokeai/app/services/model_manager_initializer.py

```diff
@@ -7,21 +7,20 @@
 from pathlib import Path
 
 import invokeai.version
+from .config import InvokeAISettings
 from ...backend import ModelManager
 from ...backend.util import choose_precision, choose_torch_device
-from ...backend import Globals
 
 # TODO: Replace with an abstract class base ModelManagerBase
-def get_model_manager(config: Args) -> ModelManager:
-    if not config.conf:
-        config_file = os.path.join(Globals.root, "configs", "models.yaml")
-        if not os.path.exists(config_file):
-            report_model_error(
-                config, FileNotFoundError(f"The file {config_file} could not be found.")
-            )
+def get_model_manager(config:InvokeAISettings) -> ModelManager:
+    model_config = config.model_conf_path
+    if not model_config.exists():
+        report_model_error(
+            config, FileNotFoundError(f"The file {model_config} could not be found.")
+        )
 
     print(f">> {invokeai.version.__app_name__}, version {invokeai.version.__version__}")
-    print(f'>> InvokeAI runtime directory is "{Globals.root}"')
+    print(f'>> InvokeAI runtime directory is "{config.root_dir}"')
```

Comment:
Seems like `root_dir` is still relying on `Globals` - see comment re: env var parsing

Reply:
This is fixed in commit cfb08d9